oauth2/oauth2-client/src/test/java/org/springframework/security/oauth2/client/registration/TestClientRegistrations.java

```diff
@@ -64,6 +64,22 @@ public static ClientRegistration.Builder clientRegistration2() {
 		// @formatter:on
 	}
 
+	public static ClientRegistration.Builder publicClientRegistrationWithNoPkce() {
```

Comment:
Please remove this as `TestClientRegistrations.clientRegistration()` can be used and the caller can simply change `builder.clientSettings(updatedClientSettings)`